NitroxServer/Communication/Packets/Processors/FootstepPacketProcessor.cs

```diff
@@ -0,0 +1,52 @@
+using NitroxModel.DataStructures.Unity;
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+
+namespace NitroxServer.Communication.Packets.Processors
+{
+    public class FootstepPacketProcessor : AuthenticatedPacketProcessor<FootstepPacket>
+    {
+        private readonly float footstepAudioRange = 20f;
```

Comment:
What about we store the range in SoundWhitelist_Subnautica.csv and get the range like in FootstepSounds_OnStep_Patch.cs#L74.
If not make it const and make a comment to reference it to Client side FootstepPacketProcessor.cs

Reply:
Done, tested